wine-tkg-git/PKGBUILD

```diff
@@ -589,6 +591,15 @@ prepare() {
 	  patch -Np1 < ../'fortnite.patch' && echo "Applied Fortnite crashfix" >> "$_where"/last_build_config.log
 	fi
 
+  # MTGA fix
+	if [ $_mtga_fix == "true" ]; then
+      if [ $_use_staging == "true" ]; then
+        patch -Np1 < ../'mtga-staging.patch' && echo "Applied MTGA crashfix" >> "$_where"/last_build_config.log
+      else
+	       patch -Np1 < ../'mtga.patch' && echo "Applied MTGA crashfix" >> "$_where"/last_build_config.log
```

Comment:
Is the indentation difference done on purpose?

Reply:
Oh no!